nextflow.config

```diff
@@ -196,6 +196,12 @@ params {
     taxpasta_add_ranklineage                = false
     taxpasta_ignore_errors                  = false
     standardisation_motus_generatebiom      = false
+
+    // Generate downstream samplesheets
+
+    // Generate downstream samplesheets
+    generate_downstream_samplesheets        = false
+    generate_pipeline_samplesheets          = "mag"
```

Comment:
Please replace this with `null` to have no default

Reply:
Done